### PR TITLE
Feature/card component

### PR DESCRIPTION
**Pull request by flaviuvr**

### CODE REVIEW

File: early-bird/src/components/home/Home.js [hunk 1]
```diff
@@ -1,17 +1,82 @@
...
 import Hello from "./hello/Hello";
 import FilterAndSort from "./filter-and-sort/FilterAndSort";
+import DisplayOffers from "./display-offers/DisplayOffers";
 
-import {useState} from "react";
+import {useState, useEffect} from "react";
 
 function Home(){
     const [offers, setOffers] = useState([]);
+    useEffect(() => {
```
Comment (cristisocaci) [inline early-bird/src/components/home/Home.js:16]: Please get the data from the backend

File: early-bird/src/components/home/display-offers/DisplayOffers.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                <Card 
+                    className="card-element"
+                    title={card.title}
+                    cost={card.cost}
+                    publisher={card.publisher}
+                    location={card.location}
+                    categories={card.categories}
+                    key={index}
+                />
+            )) : <div>Nemszobot</div>}
```
Comment (cristisocaci) [inline early-bird/src/components/home/display-offers/DisplayOffers.js]: I don't think that this line should be deployed :D

File: early-bird/src/components/home/display-offers/DisplayOffers.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.card-grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(22em, 30em));
+    justify-content: center;
+    width: 80%;
+
+    .card-element {
+        height: 11em;
+        padding: 20px;
```
Comment (cristisocaci) [inline early-bird/src/components/home/display-offers/DisplayOffers.scss:9]: You could use em here too

**Review by cristisocaci — CHANGES_REQUESTED** [review]
+ where you set padding and margin please use em

File: early-bird/src/components/navbar/Navbar.scss [hunk 4]
```diff
@@ -20,6 +20,10 @@
     display: flex;
     justify-content: center;
     align-items: center;
+
+    &:hover {
+        cursor: pointer;
+    }
 }
 #btn-right{
     display: grid;
```
Comment (cristian-nicolae) [inline early-bird/src/components/navbar/Navbar.scss:29]: LGTM!

**Review by cristian-nicolae — APPROVED** [review]

**Review by cristisocaci — APPROVED** [review]

**Pull request merged**